code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -2129,13 +2155,13 @@
 /datum/reagent/ethanol/tequilla_sunrise
 	name = "Tequila Sunrise"
 	id = "tequillasunrise"
-	description = "Tequila and orange juice. Much like a Screwdriver, only Mexican~"
+	description = "Tequila and orange juice. Much like a Screwdriver, only Mexican~."
```

Comment:
I'd remove the tilda at the end if I were you. A tilda is something used as punctuation indicating the sentence is a lewd one.

Trust me, I'm an English Major ~~Dropout~~.